config/src/config/key_manager_config.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::config::{LoggerConfig, PersistableConfig, SecureBackend};
+use anyhow::Result;
+use libra_types::account_address::AccountAddress;
+use serde::{Deserialize, Serialize};
+use std::path::{Path, PathBuf};
+
+const DEFAULT_JSON_RPC_ENDPOINT: &str = "https://127.0.0.1:8080";
+
+// Timing related defaults
+const DEFAULT_ROTATION_PERIOD_SECS: u64 = 604_800; // 1 week
+const DEFAULT_SLEEP_PERIOD_SECS: u64 = 600; // 10 minutes
+const DEFAULT_TXN_EXPIRATION_SECS: u64 = 3600; // 1 hour
+
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[serde(default, deny_unknown_fields)]
+pub struct KeyManagerConfig {
+    pub rotation_period_secs: u64,
+    pub sleep_period_secs: u64,
+    pub txn_expiration_secs: u64,
+
+    pub json_rpc_endpoint: String,
+    pub validator_account: AccountAddress,
```

Comment:
this is in secure storage, if not, we need to add it :O

crap more work for me, I think

Reply:
Thanks, David! As discussed offline, I'll come back and remove this from the config once your PR lands!